docs/platform-setup.md

```diff
@@ -18,3 +18,126 @@ For each job, disable the GitHub multibranch status:
 - Select "Disable GitHub Notifications".
 
 - Click on "Save".
+
+## Configure Nexus Docker Registry
+
+### Prerequisite
+
+- Nexus is installed.
+- NGINX Ingress Controller is installed.
```

Comment:
Is this the `jxing-nginx-ingress-controller-846d77d58b-j4vm9` pod of the `default` namespace ?
Maybe just add "in the `default` namespace"

Reply:
changes applied